src/pepr/uds-operator-config/values.schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "operator": {
+      "type": "object",
+      "properties": {
+        "UDS_DOMAIN": { "type": "string", "description": "Deprecated: Domain configuration." },
+        "UDS_ADMIN_DOMAIN": {
+          "type": "string",
+          "description": "Deprecated: Admin domain configuration."
+        },
+        "UDS_CA_CERT": { "type": "string", "description": "Deprecated: CA certificate." },
+        "UDS_ALLOW_ALL_NS_EXEMPTIONS": {
+          "type": "string",
+          "description": "Deprecated: Allow all namespace exemptions."
+        },
+        "UDS_LOG_LEVEL": {
+          "type": "string",
+          "enum": ["debug", "info", "warn", "error", "fatal"],
+          "description": "Deprecated: Log level configuration."
+        },
```

Comment:
I think this is missing trace (it is a valid log level and worked previously). Maybe not the most important because this is set directly with a zarf var on the env now (see other comment as well).